.gitignore

```diff
@@ -1,2 +1,16 @@
 cad/**
```

Comment:
let's make the title style consistent:
qa7: improve build process using Vivado

Reply:
done